pip/requirements.txt

```diff
@@ -359,6 +359,8 @@ toolz==0.7.1
 tornado==6.3.3
 tqdm==4.65.0
 traitlets==5.9.0
+# always sync version number with triton-inference-client.spec (C++ version)
+tritonclient==2.25.0
```

Comment:
@kpedro88 , this alone is not going to include python tritonclient in cmssw. Please add `py3-tritonclient` dependency in https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_14_0_X/master/python_tools.spec .